### DISCUSSION TITLE
I deleted a directory with 200+ files and it didn’t delete why is it not deleting

### DISCUSSION THREAD

**Discussion by Thunderrock424242** [community/community#65204] [Repositories] [answered]
.

**Comment by seyLu** [comment, marked as answer]
## Deleting directory via Github

### When viewing directory:
- go to top right
- click the three dots
- and click delete directory.

![delete_dir](https://github.com/community/community/assets/98249191/172bf67c-8af7-48b9-a01a-667a29edf781)

### This will bring you to a new page
- scroll down

![Screenshot from 2023-08-29 09-11-43](https://github.com/community/community/assets/98249191/fa64db31-101d-462d-94ad-5d587796153c)

### Commit and Push as usual
- write a commit message
- click commit changes

![image](https://github.com/community/community/assets/98249191/9576b702-981e-4ad5-9935-2569c8da3b2f)
 
### You should get a confirmation that the directory has been deleted
- you can also check the commits to see if the changes have been pushed to the remote repo

![image](https://github.com/community/community/assets/98249191/f58ff185-5fca-4722-926e-a30b92ebd69a)

![image](https://github.com/community/community/assets/98249191/fc868634-b3b7-44a5-8ea8-b90b97bef560)

![image](https://github.com/community/community/assets/98249191/be6eaede-e504-49e9-b74e-7b37b3549b47)

**Comment by r00ott** [comment]
I think you forget commit change
